rsts/community/roadmap.rst

```diff
@@ -6,35 +6,36 @@ Roadmap
-cases where we feel rapid prototyping would enable us to discover potential pitfalls or uncover hidden use cases we would proactively develop features, behind feature flags.
+Flyte is used actively in production at multiple companies. We pride ourselves on being extremely customer-focused and care deeply about a high quality customer experience. Thus, we always prioritize stability, reliability, observability, and maintainability over raw feature development. 
 
-Thus, it is extremely important to let the community know about your use cases and how we could adapt parts of Flyte to meet those requirements. We welcome collaboration and contributions, but please follow our Contribution Guidelines. 
+The features are developed usually in response to specific use cases and user scenarios. That being said, we are proactively thinking about the evolution of the system and how we want to keep adapting to the changing requirements. Thus, most of our changes reflect future development scenarios, and in cases where we feel rapid prototyping would enable us to discover potential pitfalls or uncover hidden use cases, we would proactively develop features, behind feature flags.
+
+Thus, it is extremely important to let the community know about your use cases and how we could adapt parts of Flyte to meet those requirements. We welcome collaboration and contributions, but please follow our Contribution Guidelines.
 
 
 Milestones and Releases
 ========================
-Flyte consists of many components and services. In true Agile fashion each service is independently iterated and co-ordiated by maintaing backwards compatible contracts using protobuf defined in `idl <https://flyte.readthedocs.io/projects/flyteidl/en/latest/>`__. Thus components like flytekit, flytepropeller, datacatalog are independently versioned.
+Flyte consists of many components and services. In true Agile fashion, each service is independently iterated and coordinated by maintaining backwards-compatible contracts using protobuf defined in `FlyteIDL <https://flyte.readthedocs.io/projects/flyteidl/en/latest/>`__. Thus components like lytekit, Flyte Propeller, and Datacatalog are independently versioned.
 
-We have decided to release a new version of the overall platform in the `flyte repo <https://github.com/flyteorg/flyte>`_ every month. Thus we create one milestone for end of every month which points to a new release of
+We have decided to release a new version of the overall platform in the `Flyte repo <https://github.com/flyteorg/flyte>`_ every month. Thus we create one milestone for end of every month which points to a new release of
 Flyte. This may change in the future, but to match our velocity of development this is our preferred option. Every release will be associated with a CHANGELOG (in the repo).
```

Comment:
```suggestion
Flyte. This may change in the future, but to match our velocity of development, this is our preferred option. Every release will be associated with a CHANGELOG (in the repo).
```